terraform.tfvars

```diff
@@ -19,7 +19,7 @@ bastion_volume_type       = "io1"
 bastion_volume_size       = 15
 bastion_iops              = 1500
 cluster_name              = "phandalin"
-rke_version               = "v1.4.4"
-kubernetes_version        = "v1.25.6-rancher4-1" # See available version https://github.com/rancher/rke/releases
-will_install_ccm          = false                # Set to true if the CCM of osc will be installed
+rke_version               = "v1.6.2"
+kubernetes_version        = "v1.30.3-rancher1-1"
```

Comment:
Isn't it v1.30.4-rancher1-1?

Reply:
I update both of them (rke%_version and k8s_version to the latest to support 1.30.5 and rke 1.6.3